client/client.go

```diff
@@ -26,12 +26,21 @@ func (e StatusError) Error() string {
 	return fmt.Sprintf("fullstory: response error: Status:%s, StatusCode:%d, RetryAfter:%v", e.Status, e.StatusCode, e.RetryAfter)
 }
 
+// DataExportClient represents an interface for interacting with the FullStory Data Export API
+type DataExportClient interface {
+	ExportList(start time.Time) ([]ExportMeta, error)
+	ExportData(id int, modifyReq ...func(r *http.Request)) (ExportData, error)
+}
+
 // Client represents a HTTP client for making requests to the FullStory API.
 type Client struct {
+	DataExportClient
```

Comment:
I don't think you want this inlined here. The `Client` itself is the one that should be implementing the interface. The type check on line 42 has you covered. Plus, calling `Client.DataExportClient.ExportList()`, for example, would panic..

Reply:
Good call. I meant to remove this.